queries/go/indents.scm

```diff
@@ -0,0 +1,17 @@
+[
+  (import_declaration)
+  (function_declaration)
+  (const_declaration)
+  (var_declaration)
+  (type_declaration)
+  (if_statement)
+  (composite_literal)
+  (func_literal)
+] @indent
+
+[
+  "("
+  ")"
+  "{"
+  "}"
```

Comment:
Indentation issue with else block:
```
if () {
} else {
    }
```
can be fixed by appending @ branch with block